logic/scripts/update_token_insight.py

```diff
@@ -366,16 +396,186 @@ def fetch_wallet_balance(wallet, label, eth_threshold):
             print (subject)
             sgw.notify_founders(body, subject)
 
-        db.session.add(contact)
-        db.session.commit()
     except Exception as e:
         print e
 
+# Fetches and stores OGN & ETH prices froom CoinGecko
+def fetch_token_prices():
+    print("Fetching token prices...")
+    
+    try:
+        url = "https://api.coingecko.com/api/v3/simple/price?ids=origin-protocol%2Cethereum&vs_currencies=usd"
+        raw_json = requests.get(url)
+        response = raw_json.json()
+
+        if "error" in response:
+            print("Error while fetching balance")
+            print(response["error"]["message"])
+            raise ValueError(response["error"]["message"])
+
+        ogn_price = response["origin-protocol"]["usd"]
+        eth_price = response["ethereum"]["usd"]
+
+        redis_client.set("ogn_price", ogn_price)
+        redis_client.set("eth_price", eth_price)
+
+        print "Set OGN price to %s" % ogn_price
+        print "Set ETH price to %s" % eth_price
+
+    except Exception as e:
+        print("Failed to load token prices")
+        print e
+
+def fetch_staked_user_count():
+    print("Fetching T3 user count...")
+    
+    try:
+        url = "https://remote.team.originprotocol.com/api/user-stats"
```

Comment:
Is that a public api with no auth? It's probably ok since I don't think the data is super sensitive and anyway will be displayed on our dashboard... I'm just asking.
Also is that URL correct? I thought it was https://investor.originprotocol.com ?

Reply:
Actually, we have two portals. https://investor.originprotocol.com and https://team.originprotocol.com We need to pull data from both. Josh commented about this above. Will include a call to investor portal too.